src/mjolnir/valhalla_add_elevation.cc

```diff
@@ -66,6 +65,7 @@ int main(int argc, char** argv) {
   const auto program = filesystem::path(__FILE__).stem().string();
   // args
   std::vector<std::string> tiles;
+  boost::property_tree::ptree config;
```

Comment:
all exe's have more or less the same changes. some still had the logging initializing stuff which I also removed.

Reply:
in fact this whole PR is mostly reverting #4220 and just restoring the way we handle things in `argparse_utils.h` based on the introduced singleton.